### DISCUSSION TITLE
Can rabbitmq.conf be used to enable feature flags?

### DISCUSSION THREAD

**Discussion by devfinwiz** [rabbitmq/rabbitmq-server#12062] [Questions] [answered]
For eg something like this: feature_flags.stream_single_active_consumer = enabled ?
The documentation does not explicitly specify or I may have missed it if we can do that or not?

I could see this around enabling feature flags: https://www.rabbitmq.com/docs/feature-flags#how-to-enable-feature-flags

**Comment by michaelklishin** [comment, marked as answer]
No, since feature flags are not node-specific configuration settings and they cannot be disabled once enabled.

They are shared cluster state. You can enable all stable feature flag with a single command on any cluster node, after all nodes have been upgraded to a version that supports that flag.

**Reply by devfinwiz** [reply]
Noted, thanks!
